# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/font-matching/wingdings-1.html
# Commit: 3b719cd12933
# Full Hash: 3b719cd1293370d843691b3656b52b04042d5ec4
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2022-03-02 04:31:47
# Regressor Bug: 1756720
# File Overlap Count: 1
# Description:
#   Bug 1756720 - When using legacy MS Symbol fonts, map character codes from the U+00xx range to the PUA range U+F0xx found in the cmap subtables. r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D139724
# ==============================================================================

diff -r 9849c6e79fa3 -r 3b719cd12933 layout/reftests/font-matching/wingdings-1.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/layout/reftests/font-matching/wingdings-1.html	Tue Mar 01 15:22:56 2022 +0000
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<style>
+body {
+  font: 16px/2 sans-serif;
+}
+span {
+  font-family: Wingdings, sans-serif;
+}
+</style>
+<body>
+Trying the Wingdings font: <span>abcdef 12345</span>
